main.c

```diff
@@ -787,7 +779,7 @@ main
     TsSupported = mutt_ts_capability();
     rootwin_set_size(COLS, LINES);
   }
-
+  
```

Comment:
Urgh!
Check your editor settings -- many have an option to kill trailing whitespace